requirements.txt

```diff
@@ -1,5 +1,6 @@
 fastapi
 uvicorn
+PyJWT
```

Comment:
do we want to put a specific version?

Reply:
All of the other deps don't have versions so I followed that norm.